examples/examples.yaml

```diff
@@ -1,6 +1,7 @@
 - path: "models/file"
   slow: ["choropleth.py"]
   no_js: ["maps_cities.py", "maps.py", "trail.py"]
+  no_diff: ["graphs.py"]
```

Comment:
it'll be a no_diff example because the networkx layout methods aren't all deterministic